src/std/main.ab

```diff
@@ -239,3 +239,39 @@ pub fun get_env_var(var: Text): Text {
 pub fun load_env_file(): Null {
     unsafe $export "\$(xargs < .env)" > /dev/null$
 }
+
+pub fun shell_isset(name: Text): Bool {
+    $[[ ! -z \$\{!name+z} ]]$ failed {
+        return false
+    }
+    return true
+}
+
+pub fun shell_constant_set(name: Text, val: Text): Bool {
+    $readonly \$name="\$val" 2> /dev/null$ failed {
+        return false
+    }
+    return true
+}
+
+pub fun shell_constant_get(name: Text): Text {
+    return unsafe $echo \$\{!name}$
+}
+
+pub fun shell_var_set(name: Text, val: Text): Bool {
```

Comment:
The same thing applies here just like for `shell_constant_set`. It should fail if name is invalid